task_5.1.py

```diff
@@ -0,0 +1,23 @@
+# Multi-paradigm programming languages, task №5.1
```

Comment:
you can leave versions without takewhile. The examples are just for reference.
I would advice you look through itertools library and try some functionality